source/MRMesh/MRObjectLoad.cpp

```diff
@@ -189,16 +190,31 @@ tl::expected<std::vector<std::shared_ptr<MR::Object>>, std::string> loadObjectFr
         else
             result = tl::make_unexpected( res.error() );
     }
-    else if ( !SceneFileFilters.empty() && filename.extension() == SceneFileFilters.front().extension.substr( 1 ) )
+    else if ( ext == u8".mru" )
     {
-        auto res = deserializeObjectTree( filename, {}, callback );
-        if ( res.has_value() )
+        auto objTree = deserializeObjectTree( filename, {}, callback );
+        if ( objTree.has_value() )
         {
```

Comment:
mb add function to load scene tree from any supported format (not to harcode extensions)